### ISSUE TITLE
API requests should use maxlag parameter

### ISSUE THREAD

**Issue by Vort** [ValterVB/VBot#4]
Using of `maxlag` parameter is an requirement for running the bot:
https://www.wikidata.org/wiki/Wikidata:Bots#Bot_accounts
https://www.mediawiki.org/wiki/Manual:Maxlag_parameter

Also, correct support for this parameter allows to get rid of 503 errors.
Here is my implementation, which may help:
https://github.com/Vort/WikiDownload/blob/master/Program.cs#L45

**Comment by ValterVB** [comment]
Yes, I know. In new versione I use it. I add the parameter `&maxlag=5` to all the API request
